python/smqtk/utils/image_utils.py

```diff
@@ -0,0 +1,54 @@
+import io
+import PIL.Image
+
+from smqtk.representation.data_element.file_element import DataFileElement
+
+
+def is_loadable_image(dfe):
+    """
+    Determine if an image is able to be loaded by PIL.
+
+    :param dfe: A data file element to check
+    :type dfe: DataFileElement
+
+    :return: Whether or not the image is loadable
+    :rtype bool:
+
+    """
+    try:
+        PIL.Image.open(io.BytesIO(dfe.get_bytes()))
+        return True
+    except IOError, ex:
+        return False
+
+
+def is_valid_element(fp, valid_content_types=None, check_image=False):
```

Comment:
This should take in a ``DataElement`` input instead of a specific file path. This opens up the use of this function with things like URLs and stuff through other element implementations.